src/components/externalLinks/ExternalLinks.tsx

```diff
@@ -480,17 +487,22 @@ function ExternalLinks({ isAppConfigView, userRole }: ExternalLinksProps) {
             {showAddLinkDialog && (
                 <AddExternalLink
                     appId={appId}
+                    isFullMode={isFullMode}
                     isAppConfigView={isAppConfigView}
                     handleDialogVisibility={handleDialogVisibility}
                     selectedLink={selectedLink}
                     monitoringTools={monitoringTools}
-                    allApps={[
-                        {
-                            label: 'All applications',
-                            value: '*',
-                            type: ExternalLinkIdentifierType.AllApps,
-                        } as IdentifierOptionType,
-                    ].concat(allApps)}
+                    allApps={
+                        isFullMode
```

Comment:
Can we replace use of ternary here?

Reply:
No. We need empty value in case it's false.